README.md

```diff
@@ -1,7 +1,28 @@
 ## Brass_Amber_BattleTowers
-Re-vamp of the old BattleTowers mod from yogbox. Developed by Brass_Herbalist, X, and Milamber
+Re-vamp of the old BattleTowers mod from yogbox. Developed by Brass_Herbalist, Milamber, X, and DerToaster
+
+Some comments in LandTower.java are from the Structure Tutorial here: https://github.com/TelepathicGrunt/StructureTutorialMod/tree/e51cca9698110559f8f586b9c71ae9a4e655d786
+by Telepathic Grunt and have be marked as such.
+
+### Current Features of 1.0.0:
+
+- Land Tower 
+- Land Tower Spawners
+- Land Tower Loot
+- Land Golem
+- Tower Crumble
+- Locked floor loot chests
+
+### Known Bugs
+
+- Land Golem enters engraged state at 1/3 of health but does not visually change
```

Comment:
This is fixed in my last commit.

Reply:
Sweet, make a pull request if you can and change the version to 1.0.1 and I can put up as a new download this afternoon.